advocacy_docs/partner_docs/VeritasGuide/03-SolutionSummary.mdx

```diff
@@ -0,0 +1,9 @@
+---
+title: 'Solution Summary'
+description: 'Brief explanation of the solution and its purpose'
+---
+NetBackup for PostgreSQL Agent extends the capabilities of NetBackup to include backup and restore of PostgreSQL databases. If a NetBackup environment is operational within an organization, then users can backup and restore EDB Postgres Advanced and EDB Postgres Extended database servers with the help of NetBackup for PostgreSQL Agent. 
```

Comment:
Should include vendor name .  Veritas Netbackup ....   see thales guide as sample.
Server should always be at end of our product names.  not separated as you have it.
Is there information on veritas site we can use?  I think we can make it more informative.

Is this their image?  It would be better if we could use the database image we have in Nutanix with the EDB logo on it.

Reply:
The information was taken from the partner site and is all the information they currently have. We can add more once we get the landing page info from the partner